draft-ietf-quic-tls.md

```diff
@@ -655,8 +662,7 @@ alerts at the "warning" level.
 
 After QUIC moves to a new encryption level, packet protection keys for previous
 encryption levels can be discarded.  This occurs several times during the
-handshake, as well as when keys are updated (see {{key-update}}).  Initial
-packet protection keys are treated specially, see {{discard-initial}}.
```

Comment:
Right now, this removed statement is still true, so I think we should retain it.

Reply:
This statement existed because previously there was an "all the keys except initial" section and an "initial keys" section. Now each flavor of keys has its own section so pointing out initial specifically here seems odd.

Reply:
I see.  That being the case, it might pay to reference all of the relevant sections here.  The statement here doesn't really reference the text that it depends on, except for key update.

Reply:
This text is in section 4.9, and the relevant sections are 4.9.1, 4.9.2 and 4.9.3 so the reader can see them coming over the horizon. Or do you mean other sections?